frontend/sw360-portlet/src/main/webapp/html/components/editRelease.jsp

```diff
@@ -72,15 +72,14 @@
     <p class="pageHeader"><span class="pageHeaderBigSpan"><sw360:out value="${component.name}"/>: <sw360:ReleaseName release="${release}" /> Edit</span>
         <span class="pull-right">
                    <core_rt:if test="${not addMode}">
-                       <input type="button" class="addButton" onclick="deleteConfirmed('' +
-                               'Do you really want to delete the release <b><sw360:ReleaseName release="${release}" /></b> ?'  +
-                               '<core_rt:if test="${not empty release.releaseIdToRelationship or not empty release.attachments}" ><br/><br/>The release <b><sw360:ReleaseName release="${release}" /></b> contains<br/><ul></core_rt:if>' +
-                               '<core_rt:if test="${not empty release.releaseIdToRelationship}" ><li><sw360:out value="${release.releaseIdToRelationshipSize}"/> linked releases</li></core_rt:if>'  +
-                               '<core_rt:if test="${not empty release.attachments}" ><li><sw360:out value="${release.attachmentsSize}"/> attachments</li></core_rt:if>'  +
-                               '<core_rt:if test="${not empty release.releaseIdToRelationship or not empty release.attachments}" ></ul></core_rt:if>', deleteRelease)"
-                              value="Delete  <sw360:ReleaseName release="${release}" /> "
-                       <core_rt:if test="${usingComponents.size()>0 or usingProjects.size()>0}"> disabled="disabled" title="Deletion is disabled as the release is used." </core_rt:if>
-                       >
+                       <input type="button" class="addButton delete-release"
+                               data-release-name="<sw360:ReleaseName release="${release}" />"
```

Comment:
Additional escaping might be necessary (see https://github.com/sw360/sw360portal/issues/534)